app/src/main/java/edu/gatech/gtorg/gitmad/threads/fragment/ThreadDetailsFragment.java

```diff
@@ -87,16 +87,19 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         /* TODO 10
          *  Create a new View.OnClickListener() for sendButton by using an anonymous inner class
          *  or a lambda expression. This anonymous inner class needs to override the onClick() method.
-
+         *
          *  Inside the onClick method, write code to tell your friends about the threads
          *  you're interested in! You don't know how the user wants to send their message,
          *  so use an implicit intent for sending messages.
-
-         * 1. Create an intent with a specified action
-         * 2. Put sendMessage in the intent as extra data
-         * 3. Set the type of intent (hint: plain text would be good here)
-         * 4. Force the app chooser to appear every time (Android Intent docs will help with this)
-         * 5. Start the activity!
+         *
+         * 1. Create a string sendMessage that creates an email message that includes the
+         *    threadName and threadDetails. I recommend the string:
+         *    String sendMessage = "About " + threadName + ": \n\n" + threadDetails;
```

Comment:
If you're going to give them the code like this, you might as well leave it outside of the TODO so they don't get confused.